Artem Sushko/Lesson7/Lesson7.Homework/Program.cs

```diff
@@ -0,0 +1,95 @@
+﻿internal class Program
+{
+    static bool Compare(string str1, string str2)
+    {
+        if (str1 == str2)
```

Comment:
The task was " but do not use build-in method", so you should implement lengh/chars comparison instead of